pkg/dao/repository_config.go

```diff
@@ -153,7 +153,8 @@ func (r repositoryConfigDaoImpl) List(
 
 	filteredDB := r.db
 
-	filteredDB = filteredDB.Where("org_id = ?", OrgID)
+	filteredDB = filteredDB.Where("org_id = ?", OrgID).
+		Joins("inner join repositories on repository_configurations.repository_uuid = repositories.uuid")
```

Comment:
Url was conditionally available, this change ensures it availability for later sorting.